README.md

```diff
@@ -490,6 +490,85 @@ To quickly copy the value of a variable into the clipboard, use the keyboard pre
+
+**Note**: Files that are GPG or EasyPG encrypted can opened and decrypted automatically by Emacs if configured appropriately. See: [Emacs Auth-source manual](https://www.gnu.org/software/emacs/manual/auth.html) for more information.
+
+The `Verb-Prelude` property may be set at the buffer level (very top of the Org document) or various header levels, and every heading level in the hierarchy with the `Verb-Prelude` property will also be loaded. If the same setting within an environment configuration file is specified at a parent and a child level, then the child will override the parent. This allows more specific or different settings to be done for a lower-level request than at a higher-level. All other file unique variables would be additive to the collection of Verb variables.
+
+Below is an example of configuring external configuration files at top buffer level, and then a different file for *Foobar Development* at lower level 2.
+
+``` org 
+#+Verb-Prelude: /path/to/prod-foobar-env.el.gpg
+
+* Foobar Blog API                    :verb:
+
+template https://foobar-blog-api.org/api/v1
+Accept: application/json
+
+** Foobar Production
+
+get /status
+
+** Foobar Development
+:properties:
+:Verb-Prelude: /path/to/dev-foobar-env.el.gpg
+:end:
+
+template https://foobar-blog-api.org/api/v2
+Accept: application/json
+
+get /status
+```
+
+In the scenario, when sending the `get /status` request for Foobar Development, `prod-foobar-env.el.gpg` is loaded first from the `#+Verb-Prelude:` buffer property. And then `dev-foobar-env.el.gpg` is loaded from properties under *`** Foobar Development`* header. The dev file would override any variables having the same key name between the two environment files. Again, all other variables would be additive. A common example would be that an *api_token* variable would likely be different between the two environments, but *user_name* would be the same for both environments. It would be logical to then have *api_token* variable specified in both, depending on environments of requests. And *user_name* specified in highest level of the Org structure to be shared among Production and Development environments.
+
+For an Emacs Lisp Prelude file, it can run any code found in the `.el` file. (Even to dynamically call and get a temporary Bearer token!) For this reason, a yes-no warning prompt is presented when loading those files unless `verb-suppress-load-unsecure-prelude-warning` is set to non-nil value. This, of course, is another reason to GPG encrypt the configuration files to help prevent configuration poisoning. Here's an example of loading `verb-var`'s using an Emacs Lisp file:
```

Comment:
```suggestion
For an Emacs Lisp Prelude file, it can run any code found in the `.el` file (even to dynamically call and get a temporary `Bearer` token!). For this reason, a yes-no warning prompt is presented when loading those files unless `verb-suppress-load-unsecure-prelude-warning` is set to non-nil value. This, of course, is another reason to GPG encrypt the configuration files to help prevent configuration poisoning. Here's an example of loading `verb-var`'s using an Emacs Lisp file:
```